scene/gui/file_dialog.cpp

```diff
@@ -67,10 +67,11 @@ void FileDialog::_native_popup() {
 	} else if (access == ACCESS_USERDATA) {
 		root = OS::get_singleton()->get_user_data_dir();
 	}
+	String tr_title = atr(get_title());
```

Comment:
Apparently this shadows `tr_title` from Window.

Two things then:
- This could likely use `tr_title` directly?
- But to do it properly it would likely make sense to make `tr_title` private and add a getter like `get_translated_title`.

Reply:
If we do decide to add a getter, then this can be simplified too:
```
main/viewport.cpp
333:            TextLine title_text = TextLine(p_window->atr(p_window->get_title()), title_font, font_size);
```

Reply:
Added `get_translated_title`.